h5bp/web_performance/compression.conf

```diff
@@ -39,6 +39,7 @@ gzip_types
   application/atom+xml
   application/geo+json
   application/javascript
+  application/x-javascript
```

Comment:
This is an old and deprecated MIME type for `application/javascript`.

Reply:
Yes,  But still most of the people are using this we are also. so I assume we should add this.